components/bl-AnimateOnScroll-component/README.md

```diff
@@ -0,0 +1,27 @@
+# Animate on Scroll
+
+Animate on Scroll is a component of Backendless UI-Builder designer. This component allows adding animation for elements when scrolling the page.
+
+The component based on external [Animate on Scroll](https://github.com/michalsnik/aos).
+
+## Properties
+
+| Property       | Type                                                                                                                                                                                                                                                                                                                                                  | Default Value | Logic             | Data Binding | UI Setting | Description                                                                                                       |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-------------------|--------------|------------|-------------------------------------------------------------------------------------------------------------------|
+| Disable For    | *Select* <br/>[`mobile`, `tablet`, `all`, `none`]                                                                                                                                                                                                                                                                                                     | "false"       | Disable For Logic | YES          | YES        | Allows to determine whether or not to disable the animation                                                       |
```

Comment:
`false` is the default value for `Disable For`? but in the available list there is only: [`mobile`, `tablet`, `all`, `none`]